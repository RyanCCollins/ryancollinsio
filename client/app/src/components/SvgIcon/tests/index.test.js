import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';
import SvgIcon from '../index';

describe('<SvgIcon />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <SvgIcon />,
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
