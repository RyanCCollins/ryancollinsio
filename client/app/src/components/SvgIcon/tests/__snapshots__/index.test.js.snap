exports[`<SvgIcon /> should render with default props 1`] = `
<svg
  viewBox="0 0 24 24" />
`;
